### PR TITLE
Made custom user model,  added JWT authentication

### PR DESCRIPTION
**Pull request by oliabn**
Made custom user model, custom user manager, and admin, and added JWT authentication.

### CODE REVIEW

File: backend/accounts/admin.py [hunk 1]
```diff
@@ -1 +1,29 @@
-# Register your models here.
+from django.contrib import admin
+from django.contrib.auth.admin import UserAdmin
+from django.utils.translation import gettext_lazy as _
```
Comment (approximator) [inline backend/accounts/admin.py:3]: Is this needed here?

File: backend/accounts/urls.py [hunk 2]
```diff
@@ -5,25 +5,25 @@
...
     # session authentication
     # path("api/session/auth/", include("rest_framework.urls", namespace="rest_framework")),
 
     # Token authentication
     # api/auth/users/
-    path(r'api/auth/', include('djoser.urls')),
-
-    # api/auth/token/  login, logout, users, user/<int:id>/
-    re_path(r'^api/auth/', include('djoser.urls.authtoken')),
+    # path(r'api/auth/', include('djoser.urls')),
```
Comment (approximator) [inline backend/accounts/urls.py:26]: Please remove unneded code

**Review by approximator — CHANGES_REQUESTED** [review]
Format code:

```python
python .github/scripts/python_style.py --fix
```

File: backend/config/settings.py [hunk 3]
```diff
@@ -119,16 +123,60 @@
...
+        # 'rest_framework.authentication.TokenAuthentication',
 
         # # session authentication
         # 'rest_framework.authentication.BasicAuthentication',
         # 'rest_framework.authentication.SessionAuthentication',
     ],
 }
 
+SIMPLE_JWT = {
+    "ACCESS_TOKEN_LIFETIME": timedelta(minutes=5),
```
Comment (approximator) [inline backend/config/settings.py:142]: I think, 5 minutes lifetime is too short.

Comment (oliabn) [reply backend/config/settings.py]: Changed timedelta(day=1)

**Review by approximator — APPROVED** [review]

**Pull request merged**